plugins/action_update_review_assignments/plugin.json

```diff
@@ -3,8 +3,9 @@
   "code": "updateReviewAssignmentsStatus",
```

Comment:
Also might get the `status` at the end of this folder

Reply:
Sure, I was thinking the folder names were getting a bit long, but it's no biggie.